taxi/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+from taxi_service.settings import AUTH_USER_MODEL

```

Comment:
it's better to import settings in this way
`from django.conf import settings`
And import constants from these settings because it defines global settings as you may have other settings files in your project